package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "lab-07-cowsay-http-server",
+  "version": "1.0.0",
+  "description": "401 JS --  Lab 07 cowsay server ===",
+  "main": "index.js",
+  "scripts": {
+    "nodemon": "watch:  ./node_modules/nodemon/bin/nodemon.js server.js",
```

Comment:
Your script to run nodemon is inaccurate. It's OK if you want to use the word ``nodemon`` but the path name shouldn't have ``watch``. Change this to 

```
"nodemon": "./node_modules/nodemon/bin/nodemon.js server.js",
```